notebooks/search/03-ELSER.ipynb

```diff
@@ -39,7 +41,7 @@
    "source": [
     "# Create Elastic Cloud deployment\n",
     "\n",
-    "If you don't have an Elastic Cloud deployment, sign up [here](https://cloud.elastic.co/registration?fromURI=%2Fhome) for a free trial.\n",
+    "If you don't have an Elastic Cloud deployment, sign up [here](https://cloud.elastic.co/registration?utm_source=github&utm_content=elasticsearch-labs-notebook?fromURI=%2Fhome) for a free trial.\n",
```

Comment:
two question marks. remove the fromURI param.

Reply:
I might have added `fromURI` param in multiple instance. Will update

Reply:
added in 5725f74106dc37b600e6ba7f4f4433ad20e7bb7c